### PR TITLE
net5 TFM

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/StarWars/StarWarsSchema.cs [hunk 1]
```diff
@@ -9,8 +9,10 @@ public class StarWarsSchema : Schema
         public StarWarsSchema(IServiceProvider provider)
             : base(provider)
         {
-            Query = provider.GetRequiredService<StarWarsQuery>();
-            Mutation = provider.GetRequiredService<StarWarsMutation>();
+            Query = (StarWarsQuery)provider.GetService(typeof(StarWarsQuery)) ?? throw new InvalidOperationException();
+            Mutation = (StarWarsMutation)provider.GetService(typeof(StarWarsMutation)) ?? throw new InvalidOperationException();
```
Comment (Shane32) [inline src/StarWars/StarWarsSchema.cs:13]: Why?

**Review by Shane32 — APPROVED** [review]

File: src/StarWars/StarWarsSchema.cs (hunk 1 above)
Comment (sungam3r) [reply src/StarWars/StarWarsSchema.cs:13]: Long time ago we removed `GetRequiredService` extension method from GraphQL.NET codebase.

File: src/AspNetCoreMulti/Example/Startup.cs [hunk 2]
```diff
@@ -20,13 +20,13 @@ public void ConfigureServices(IServiceCollection services)
...
-            .AddErrorInfoProvider(opt => opt.ExposeExceptionStackTrace = true)
-            .AddSystemTextJson()
-            .AddUserContextBuilder(httpContext => new GraphQLUserContext { User = httpContext.User });
+#pragma warning disable CS0612 // Type or member is obsolete
+            services
+                .AddGraphQL(options => options.EnableMetrics = true)
+                .AddErrorInfoProvider(opt => opt.ExposeExceptionStackTrace = true)
+                .AddSystemTextJson()
+                .AddUserContextBuilder(httpContext => new GraphQLUserContext { User = httpContext.User });
+#pragma warning restore CS0612 // Type or member is obsolete
```
Comment (Shane32) [inline src/AspNetCoreMulti/Example/Startup.cs:29]: Should use non-obsolete methods.

Comment (sungam3r) [reply src/AspNetCoreMulti/Example/Startup.cs:29]: I'll change usages after April release.

**Pull request merged**